templates/base/head_navbar.tmpl

```diff
@@ -105,9 +105,20 @@
 					<span class="only-mobile">{{ctx.Locale.Tr "create_new"}}</span>
 				</span>
 				<div class="menu">
-					<a class="item" href="{{AppSubUrl}}/repo/create">
-						{{svg "octicon-plus"}} {{ctx.Locale.Tr "new_repo"}}
-					</a>
+					{{if .SignedUser.CanCreateRepo}}
+						<a class="item" href="{{AppSubUrl}}/repo/create">
+							{{svg "octicon-plus"}} {{ctx.Locale.Tr "new_repo"}}
+						</a>
+					{{else}}
+						<a
+							class="disabled-with-tooltip item"
+							data-tooltip-content="{{ctx.Locale.TrN .SignedUser.MaxCreationLimit "repo.form.reach_limit_of_creation_1" "repo.form.reach_limit_of_creation_n" .SignedUser.MaxCreationLimit}}"
+							href="/"
+							onclick="event.stopPropagation(); event.preventDefault()"
```

Comment:
There should no inline-script, otherwise it makes CSP harder. https://github.com/go-gitea/gitea/issues/305

Reply:
I might not be able to get around to this for a while. Feel free to make any necessary adjustments.